test/server/utility.h

```diff
@@ -11,14 +11,21 @@
 
 namespace Envoy {
 namespace Server {
-namespace {
 
 inline envoy::config::listener::v3::Listener parseListenerFromV3Yaml(const std::string& yaml) {
   envoy::config::listener::v3::Listener listener;
   TestUtility::loadFromYamlAndValidate(yaml, listener);
   return listener;
 }
 
-} // namespace
+inline std::string testDomainSocketName() {
```

Comment:
I think instead of this, you should use `TestEnvironment::temporaryPath(TestUtility::uniqueFilename());`. There's not a good reason to use the abstract namespace in these tests.

Reply:
Added a comment explaining why this not that.